Documentation/gettingstarted/k8s-install-azure.rst

```diff
@@ -102,3 +104,14 @@ Limitations
 
 * All VMs and VM scale sets used in a cluster must belong to the same resource
   group.
+
+.. _azure_troubleshooting:
+
+Troubleshooting
+===============
+* If ``kubectl exec`` to a pod fails to connect, restarting ``tunnelfront`` POD may help.
+* PODs may fail to gain a ``.spec.hostNetwork`` status even if restarted and managed by Cilium.
+* If some connectivity tests fail to reach the ready state you may need to restart the unmanaged pods again.
+* Some connectivity tests may fail. This is being tracked at `a Cilium GitHub issue
+  <https://github.com/cilium/cilium/issues/12113>`_.
```

Comment:
Maybe mention the issue number in the link text?
```suggestion
* Some connectivity tests may fail. This is being tracked in `Cilium GitHub issue
  #12113 <https://github.com/cilium/cilium/issues/12113>`_.
```

Reply:
Applied, thanks!